### ISSUE TITLE
Inspector hangs when running offline (on Win7)

### ISSUE THREAD

**Issue by seanpoulter** [node-inspector/node-inspector#645]
With the node-inspector started with the PC offline:
> \>node-inspector
> Node Inspector v0.10.1
> Visit http://127.0.0.1:8080/debug?ws=127.0.0.1:8080&port=5858 to start debugging.

starting a process to debug hangs:
> \>node --debug node_modules\karma-cli\bin\karma start
> Debugger listening on port 5858

The served page loads the inspector but not the debugging data and sources. Checking the console for the node-inspector shows:
> Request with id = 10 failed. "ErrorNotConnected: Debugged process exited."
>        InspectorBackendClass.AgentPrototype.dispatchResponse @ InspectorBackend.js:958
>        InspectorBackendClass.Connection.dispatch @ InspectorBackend.js:482
>        InspectorBackendClass.WebSocketConnection._onMessage @ InspectorBackend.js:690

Environment:
node v0.12.2 x64 on Windows 7 x64, running without an active network connection
<br/>
Please let me know any next steps to troubleshoot this. It's quite an inconvenience.

**Comment by 3y3** [comment]
How about
```
>node --debug-brk node_modules\karma-cli\bin\karma start
```

**Comment by seanpoulter** [comment]
When running the following offline:
```
(other console)>node-inspector
>node --debug-brk node_modules\karma-cli\bin\karma start
```
the same behaviour shows. Opening the served inspector in the browser hasn't loaded the source files. Although the Chrome console (for the inspector) does show an additional error id 25:

>Request with id = 10 failed. "ErrorNotConnected: Debugged process exited."
>&nbsp;&nbsp;&nbsp;&nbsp;InspectorBackendClass.AgentPrototype.dispatchResponse @ :8080/sdk/InspectorBackend.js:958
>Request with id = 25 failed. "Error: Not implemented."
>&nbsp;&nbsp;&nbsp;&nbsp;InspectorBackendClass.AgentPrototype.dispatchResponse @ :8080/sdk/InspectorBackend.js:958

**Comment by 3y3** [comment]
Oh, stop! You can reproduce this only if you don't have any network connections?

**Comment by seanpoulter** [comment]
Yes 3y3, it works great otherwise. This only is a problem when the PC is not connected, as an example when the wifi drops. I was expecting this to work locally and not require the internet. Should it?

**Comment by 3y3** [comment]
I have the same error. This is a nodejs error in dns module. In other words dns can't resolve localhost in debugger connection when there is no connections.
This was fixed in iojs, but exists in 0.12.
To fix this problem on my machine I installed `microsoft loopback adapter`.

**Comment by seanpoulter** [comment]
OK, thanks for confirming @3y3.

I opened an issue for node yesterday for dns resolvning localhost: joyent/node/25338, I just didn't know how to workaround it to confirm it was the same for this.
